processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -149,6 +150,17 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
+  public double estimateSelectivity(ColumnSelector columnSelector, BitmapIndexSelector indexSelector)
+  {
+    // Estimate selectivity with attribute value independence assumption
```

Comment:
This assumption could be "checked" approximately the same way as in `computeNonOverlapRatioFromFirstNBitmapSamples()`